sample-testable-package/Tests/SampleTestablePackageTests/SampleTestablePackageTests.swift

```diff
@@ -1,4 +1,5 @@
 import XCTest
+
```

Comment:
Really? A spurious newline?

Reply:
I actually can't find out why this wasn't reported before. This is needed based on how swift-format behaves.

Did some tests locally, it would sometimes randomly emit a warning about this, and sometimes not.
I couldn't find out what makes it care about that line, and what makes it not. Tried different configurations as well 
...

Reply:
I'm strongly against needing the newline, frankly... If `swift-format` is demanding it, to me that's a bug in `swift-format`.

Reply:
Yeah, obviously it does but to me it's visually jarring and completely pointless to have that gap. Why should `@testable` imports be separated?

(I know, I know, `swift-format` isn't gonna change... but there should really be more control.)